### PR TITLE
#178 analytics

### PR DESCRIPTION
**Pull request by jorre127**

### CODE REVIEW

File: ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 50;
```
Comment (vanlooverenkoen) [inline ios/Runner.xcodeproj/project.pbxproj:6]: what is this exactly? Or why has this changed?

File: lib/navigator/main_navigator.dart [hunk 2]
```diff
@@ -40,7 +42,7 @@ class MainNavigatorWidget extends StatefulWidget {
 
 class MainNavigatorWidgetState extends State<MainNavigatorWidget> with MainNavigationMixin {
   static final GlobalKey<NavigatorState> _navigationKey = GlobalKey<NavigatorState>();
-  static final List<NavigatorObserver> _navigatorObservers = [];
+  static final List<NavigatorObserver> _navigatorObservers = [GetIt.I.get<FireBaseAnalyticsRepository>().routeObserver];
```
Comment (vanlooverenkoen) [inline lib/navigator/main_navigator.dart:45]: add extra formatting with `,`

File: lib/repository/analytics/firebase_analytics_repository.dart [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  @override
+  RouteObserver<Route> get routeObserver => FirebaseAnalyticsObserver(analytics: _analytics);
+
+  _FireBaseAnalyticsRepository(this._analytics);
+
+  @override
+  void init() {
+    _analytics.logAppOpen();
+    WidgetsBinding.instance!.addObserver(this);
```
Comment (vanlooverenkoen) [inline lib/repository/analytics/firebase_analytics_repository.dart:26]: don't forget to remove the observer in some kind of dispose method

**Review by vanlooverenkoen — CHANGES_REQUESTED** [review]

File: ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jorre127) [reply ios/Runner.xcodeproj/project.pbxproj:6]: Da is blijkbaar een indicator van u project versie. ObjectVersion 50 Is compatible vanaf Xcode 9.3

**Review by NicolaVerbeeck — APPROVED** [review]

**Review by vanlooverenkoen — APPROVED** [review]

**Pull request merged**